source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,127 @@
+
+### Script names
+
+The name of the script here is `myfirsttask`. It's at the top of the indentation. Each top-level indentation entry is a separate script. Here is an example of two different scripts:
+
+```dscript_green
+myfirsttask:
+    type: task
+    script:
+    - narrate "This is is task number one!"
+
+mysecondtask:
+    type: task
+    script:
+    - narratee "This is task number two!"
+```
+
+This example demonstrates what having two different scripts in the same file looks like. `myfirsttask` and `mysecondtask` are two different scripts, and each is a fully self-contained script that can be run with the `run` command, either via the `/ex` command in-game or in a Denizen script.
+
+### Script types
+
+Below the script name - on the second level of indentation - you'll see the `type` and `script` entries.
+
+The `type` option is where you specify which `type` of Denizen script you're writing. In this case, we're writing a `task` script. You may have seen `world` scripts, `item` scripts, `inventory` scripts, and *maybe* even a `procedure` script. For now, let's focus on `task` scripts. Make sure you write `type: task` in your script on the second level of indentation, below the script name, like this:
+
+```dscript_blue
+myfirsttask:
+    type: task
+```
+
+### Script commands
+
+Below the `type` entry, you'll see the `script` entry. This is where you'll be doing most of your work in Denizen! Under the `script` entry is where you write the content of the script - no matter which kind of script it is, the `script` entry is where you tell Denizen what to do.
+
+Let's look at another example script:
+
+```dscript_green
+myfirsttask:
+    type: task
+    script:
+    - narrate "This is a valid task script!"
+```
+
+This script only has one `command` in it, the `narrate` command. As a reminder, `narrate` will display the specified text to the attached player. You can also specify a custom list of targets to `narrate` to if you want to `narrate` to multiple players!
+
+Notice how the text we want to narrate - `This is a valid task script!` - is enclosed in `""` double quotes. Because the message we want to narrate has ` ` spaces in it, we enclose it in double quotes to make sure that the command only has one `argument`. Denizen `commands` have `arguments` that are separated by ` ` spaces. For example, `run one two three` has arguments `one`, `two`, and `three`, but `run "one two three"` only has one argument, `one two three`.
+
+# Completed product
+
+At this point, you should have a task script that's ready to run! Go on, give it a try - use the `/ex run (YourTaskName)` command to run the task in-game. For example, the in-game command to run the above script would be `/ex run myfirsttask`.
+
+If you see the text that you wrote for the `narrate` command, you've successfully written your first task script. Congratulations!
+
+TKTK SCREENSHOT (in-game script successfully run)
```

Comment:
Interesting typo... `TODO`?

Reply:
https://en.wikipedia.org/wiki/To_come_(publishing)

Reply:
Is it `TK` or is it `TODO`? Cause that's neither of them.

Reply:
Many authors - I among them - use a double TK so that it can be searched without returning false positives.